C_vs_Fortran_address_errors

```diff
@@ -0,0 +1,43 @@
+fine_array[81,10]=5.000000 at 0x7f44bc034618
```

Comment:
Before merging, this type of debugging note file should be removed.

Reply:
For sure.  This kind of cruft was my lazy way of moving notes across environments.